src/submission/repositories/xpub-submission-root.ts

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import * as Knex from 'knex';
+import { InfraLogger as logger } from '../../logger';
+import { SubmissionId } from '../types';
+import { SubmissionRepository, SubmissionDTO } from './types';
+
+type entryMeta = {
+    articleType: string;
+    title: string;
+};
+// this is the xpub schema type
+type DatabaseEntry = {
```

Comment:
Are we sure this type shouldn't live somewhere else? :thinking: ... not saying we should move it - just a thought.

Reply:
It's internal to the repository because its the type def of xpubs schema. The idea here is that nothing outside of the repository should know about the model of the db schema so this type should by design never need to be accessed externally

Reply:
This is going to be unique to the repo thought is it not? It will also not be used anywhere else so I would say it should live here.